newrelic/resource_newrelic_notifications_channel.go

```diff
@@ -0,0 +1,175 @@
+package newrelic
+
+import (
+	"context"
+	"fmt"
+	"github.com/newrelic/newrelic-client-go/pkg/ai"
+	"github.com/newrelic/newrelic-client-go/pkg/notifications"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+)
+
+var notificationsChannelTypes = map[notifications.AiNotificationsChannelType][]string{
+	"EMAIL":                         {},
```

Comment:
Did these types get generated in the client? If so, let's reference the types in the client :)